enterprise_extensions/chromatic/README.md

```diff
@@ -12,14 +12,16 @@ n_earth = SW.ACE_SWEPAM_Parameter()('n_earth')
 sw = SW.solar_wind(n_earth=n_earth)
 mean_sw = deterministic_signals.Deterministic(sw, name='mean_sw')
 ```
-3. There is also a function to create a Fourier design matrix for constructing a Gaussian process. This can be used as desired, but works best as a *perturbation* to the mean signal defined above.
```

Comment:
We probably don't need to enforce style in a README.  If we manually undo this change, will it cause a lint error?

Reply:
I don't think it will. I think I was a bit draconian with `autopep8`. `flake8` should only actually check `.py` files.